.github/workflows/ci.yaml

```
@@ -89,6 +89,16 @@ jobs:
           CUDA_VER: "11.2.2"
           CENTOS_VER: "7"
 
+        - DOCKERIMAGE: linux-anvil-cuda
+          DOCKERTAG: "11.3"
+          CUDA_VER: "11.3.1"
+          CENTOS_VER: "7"
+
+        # - DOCKERIMAGE: linux-anvil-cuda
+        #   DOCKERTAG: "11.4"
+        #   CUDA_VER: "11.4"
+        #   CENTOS_VER: "7"
```

The 11.4 image is now up :)

```suggestion
        - DOCKERIMAGE: linux-anvil-cuda
          DOCKERTAG: "11.4"
          CUDA_VER: "11.4"
          CENTOS_VER: "7"
```

---

Oh, nice! Thanks for the hint @jakirkham!